Snippets/Core/Core_6/UpgradeGuides/5to6/UpgradeMessageHandler.cs

```diff
@@ -37,4 +37,88 @@ await context.Publish(new MyEvent())
         }
     }
     #endregion
+
+
+    #region 5to6-handler-migration-beginning
+    public class MigrationBeginning : IHandleMessagesFromPreviousVersions<MyMessage>
+    {
+        public IBus Bus { get; set; }
+
+        public void Handle(MyMessage message)
+        {
+            Bus.Send(new MyOtherMessage());
+            Bus.Publish(new MyEvent());
+        }
+    }
+    #endregion
+
+    #region 5to6-handler-migration-step1
```

Comment:
Since the previous snippet shows the entire class, do we need to repeat the entire code again? Why not just have the new Handle method enclosed in a region? I think that'd be much better to read.